frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue

```diff
@@ -1733,6 +1741,10 @@ export default Vue.extend({
 
     this.setupHotkeys(this.hotkeyListenerScope)
 
+    if ( this.is_active ) {
```

Comment:
Once image annotation components are mounted, need to activate registered hotkeys ofr the active one.